CppCoreGuidelines.md

```diff
@@ -7152,16 +7152,16 @@ Some of these expressions are unconditionally bad (e.g., they rely on undefined
 
 **Example**:
 
-	x=k*y+z;	// OK
+	x=k*y+z;              // OK
 
-	auto t1 = k*y;	// bad: unnecessarily verbose
+	auto t1 = k*y;        // bad: unnecessarily verbose
 	x = t1+z;
 
-	if(0<=x && x<max) // OK
+	if(0<=x && x<max) {}  // OK
```

Comment:
My earlier comment was about adding these empty statements still apply.  Not all examples are intended to be copied straight into code without modification or completion.  A good way to make that apparent to just leave certain statements incomplete.  It also allows us to draw attention to the essential point.